Whoaverse/Whoaverse/Controllers/CommentController.cs

```diff
@@ -344,11 +344,11 @@ public ActionResult SubmitComment()
                 {
                     // notify comment author that his comment has been deleted by a moderator
                     MesssagingUtility.SendPrivateMessage(
-                        "Whoaverse",
+                        "Voat",
                         commentToDelete.Name,
                         "Your comment has been deleted by a moderator",
                         "Your [comment](/v/" + commentSubverse + "/comments/" + commentToDelete.MessageId + "/" + commentToDelete.Id + ") has been deleted by: " +
-                        "[" + User.Identity.Name + "](/u/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
+                        "[" + User.Identity.Name + "](/user/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
```

Comment:
Okay, but the issue still remains. I think @PuttItOut could help look into this or maybe you could too, I think this is the relevant area: https://github.com/voat/voat/blob/master/Whoaverse/Whoaverse/Utils/Components/ContentFilters.cs#L51